frame/clearing-house/src/tests/close_position.rs

```diff
@@ -234,6 +236,111 @@ fn should_realize_short_position_funding() {
 	});
 }
 
+#[test]
+fn should_fail_if_pushes_index_mark_divergence_above_threshold() {
+	let config = MarketConfig { taker_fee: 0, ..Default::default() };
+
+	with_trading_context(config, as_balance(1_000_000), |market_id| {
+		// Set maximum divergence to 10%
+		set_maximum_oracle_mark_divergence((10, 100).into());
+
+		let vamm_id = &get_market(&market_id).vamm_id;
+		OraclePallet::set_price(Some(100)); // 1 in cents
+		VammPallet::set_price_of(vamm_id, Some(1.into()));
+
+		// Alice opens a position (no price impact)
+		assert_ok!(TestPallet::open_position(
+			Origin::signed(ALICE),
+			market_id,
+			Long,
+			as_balance(1_000_000),
+			as_balance(1_000_000),
+		));
+
+		// Alice tries to close her position, but it fails because it pushes the mark price too
+		// below the index Closing tanks mark to 89% of previous price
```

Comment:
Nitpick: Missing a period in '... index **.** Closing tanks ...' ?

Reply:
nice catch

Reply:
yep